dandiapi/api/tests/test_dandiset.py

```diff
@@ -694,6 +694,37 @@ def test_dandiset_rest_create_embargoed(api_client, user):
     }
 
 
+@pytest.mark.parametrize(
+    ('embargo_status', 'user_status', 'success'),
+    [
+        (Dandiset.EmbargoStatus.OPEN, 'owner', False),
+        (Dandiset.EmbargoStatus.OPEN, 'anonymous', False),
+        (Dandiset.EmbargoStatus.OPEN, 'not-owner', False),
+        (Dandiset.EmbargoStatus.EMBARGOED, 'owner', True),
```

Comment:
Personally I think this is fine, since this test is checking the combination of `embargo_status` and `user_status`. The fact that only one happens to be `True` doesn't really matter imo.